src/components/ContributorShowcase/ContributorShowcase.jsx

```diff
@@ -0,0 +1,40 @@
+import { StaticQuery, graphql } from 'gatsby';
+import React from 'react';
+import styles from './ContributorShowcase.module.css';
+
+function getContributors(data) {
+  const cont = [];
+  data.contributorsJson.contributors.forEach((item) => {
+    cont.push(
+      <div className={styles.griditem} key={item.name}>
+        <img className={styles.image} alt={item.profile} src={item.avatar_url} />
+        <a href={item.profile}>{item.name}</a>
+      </div>,
+    );
+  });
+  return cont;
+}
+
+function ContributorShowcase() {
+  return (
+    <div>
+      <h2>Contributors</h2>
+      <br />
+      <StaticQuery
+        query={graphql`
+          query queryJSON {
+            contributorsJson {
```

Comment:
nit. can we call this `contributorsJSON` instead?

Reply:
as far as my knowledge of gatsby-transformer-json goes, there is a very specific naming scheme to read json files, believe me I wouldn't have named it in camelcase if I didn't have to, but I'll try (no promises).

Reply:
Oooooooh I see! Didn't think about that. Makes sense.
If you can't change it no worries. Very glad you care about naming! 😏